extradict/nested_data.py

```diff
@@ -43,12 +43,19 @@ def __len__(self):
         return len(self.data)
 
     def __repr__(self):
+        def indent_content(r):
+            if "\n" not in r:
+                return r
+            offset = len(r) - len(r.lstrip("[{"))
+            r = r[:offset] + ("\n" if r[offset] != "\n" else "") + indent(r[offset:], "    ")
```

Comment:
Why did you not use the `indent_content()` function here too?

Reply:
Take a look at line 46: because this _is_ the  "indent_content" function. :-)

Reply:
Aaaaah :-)